src/components/organisms/ItemForm/ItemFormContainer.jsx

```diff
@@ -42,5 +44,16 @@ export default compose(
           history.push(`/${collectionsRoute}/${activeItem['name']}?ref=${branch}`)
         })
     }
+  }),
+  withHandlers({
+    openModal: ({ toggleNameModal, cancelRename, editFileName, setRenameFile, match }) => (e) => {
+      const { value } = e.target
+      if (value) {
```

Comment:
@tbrandle I think we should avoid checking for empty string with just value here, would be best to do `value.trim() !== ''` or a similar comparison to make sure it isn't completely empty, or just whitespace